sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/plans/SQLHelper.scala

```diff
@@ -83,4 +85,11 @@ trait SQLHelper {
       }
     }
   }
+
+  def getSparkHome(): String = {
```

Comment:
I think you could even make it a `lazy val` can call it `sparkHome` I guess but shouldn't be a big deal.